src/components/container/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import App from '../app';
+import EasterEgg from '../easter-egg';
+import getPipelineData from '../../utils/data-source';
+import './container.css';
+
+/**
+ * Top-level component for the use-case where Kedro-Viz is run as a standalone
+ * app rather than imported as a library/package into a larger application.
+ */
+const Container = () => (
```

Comment:
Optional: I might've preferred to call `AppContainer` just to allow this to be more specific.

Reply:
Maybe even `KedroViz` as that's the alias being used? Makes sense based on intended usage, since that's the public component.

Reply:
Yeah that's a good shout! I'm inclined to leave it for now because we'll be revisiting/refactoring this when we deprecate the library/package use-case, but can revisit if you feel strongly about it :)